.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.3.0
+    hooks:
+      - id: ruff
+        args: [--fix, --exit-non-zero-on-fix]
+      - id: ruff-format
+
+  # - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
You can remove this commented out Prettier code here, @kush34 :) We're not using it in Scribe-Data.